### ISSUE TITLE
error

### ISSUE THREAD

**Issue by Hosseingsd** [luckas72/Anki_AddonCode_Exporter_Addon#1]
When I click on the export button this appears:
![issue](https://user-images.githubusercontent.com/110593678/184532360-190546f3-5380-437d-917e-8f94596110dd.png)

**Comment by luq7** [comment]
Do you have a directory to `C:\P\addons21`? 
Could you please reopen the app to see if it disappears?

**Comment by Hosseingsd** [comment]
> C:\P\addons21

No, I don't have that. Never see any P folder in my C drive actually!

> Could you please reopen the app to see if it disappears?

I reopened Anki in many ways! Like disabling most addons, removing and adding AddonCode_Exporter_Addon again, and even restarting my laptop. After non of this helps, I posted here.

**Comment by luq7** [comment]
That is very strange. Have you tried the [executable version](https://github.com/luckas72/Anki_Addon_Exporter) of the addon?

**Comment by Hosseingsd** [comment]
> the executable version

I have no idea about what it is (I don't have programming knowledge)


Actually, I remember that I used this addon before and it worked well. I'm not sure but I guess it was before the latest update.

**Comment by luq7** [comment]
Yeah, so I have a stand-alone version of the exporting tool. It seems working on my side. However, I am using Mac. So it could be the update in Windows that triggered the error.
